tests/code/test_minor.py

```diff
@@ -546,3 +547,64 @@ def test_getSummerTerms():
         assert len(list(noSummerTerms)) == 0
 
         transaction.rollback()
+        
+        
+@pytest.mark.integration
+def testDeclareMinorInterest():
+    
+    with mainDB.atomic() as transaction:
+        # Get three students with interest in minor
+        student1 = User.get_by_id("agliullovak")
+        student2 = User.get_by_id("partont")
+        student3 = User.get_by_id("bryanta")
+        
+        assert student1.declaredMinor == False
+        assert student2.declaredMinor == False
+        assert student3.declaredMinor == False
+        
+        # Declare students interested in minor
+        declareMinorInterest("agliullovak")
+        declareMinorInterest("partont")
+        declareMinorInterest("bryanta")
+        
+        student1 = User.get_by_id("agliullovak")
+        student2 = User.get_by_id("partont")
+        student3 = User.get_by_id("bryanta")
+        
+        assert student1.declaredMinor == True
+        assert student2.declaredMinor == True
+        assert student3.declaredMinor == True
+        
+        transaction.rollback()
+
+
+@pytest.mark.integration
+def testGetDeclaredMinorStudents():
+    
+    with mainDB.atomic() as transaction:
+        # Get all the declared students
+        declaredStudents = getDeclaredMinorStudents()
+        
+        assert declaredStudents == []
+        assert len(declaredStudents) == 0
+        
+        student1 = User.get_by_id("agliullovak")
+        student2 = User.get_by_id("partont")
+        student3 = User.get_by_id("bryanta")
+        
+        student1.declaredMinor = not student1.declaredMinor
```

Comment:
We should just probably assert these are False initially and then just set them to True directly.